packages/cxl-lumo-styles/scss/cxl-vaadin-button.scss

```diff
@@ -1,3 +1,5 @@
+// @see https://github.com/vaadin/vaadin-button/blob/master/theme/lumo/vaadin-button-styles.html
```

Comment:
Also add `https://github.com/vaadin/vaadin-button/issues/142` at the top for rationale.